app/modules/entities/components/layouts/author_display.svelte

```diff
@@ -33,6 +37,9 @@
       </span>
     {/if}
   </div>
+  {#if image.url}
+    <img src={imgSrc(image.url, 56)} alt={i18n('author picture')} />
+  {/if}
```

Comment:
Any reason to move the image to the block end? Keeping it to the block start preserves the consistency with other places such as search results/claim autocomplete, and makes, imo, authors lists easier to read, as:
- the eye can lazily look at images, and start reading the associated text only if necessary (thus the idea to have images appear first in the row)
- that text being nearby (rather than separated by a the widest margin possible), the eye doesn't have to move too much, reducing fatigue

Reply:
commit is now dropped, since it can be discussed in #407